app/helpers/provider/admin/dashboards_helper.rb

```diff
@@ -19,22 +19,29 @@ def sign_class(widget)
     widget.percentual_change > 0 ? 'u-plus' : 'u-minus'
   end
 
-  def dashboard_collection_link(singular_name, collection, path, plural = nil, icon_name = nil)
-    link_to path, class: 'DashboardNavigation-link' do
-      link_text = pluralize(number_to_human(collection.size), singular_name, plural)
+  def dashboard_navigation_link(link_text, path, icon_name: nil, notice: false)
+    class_name = "DashboardNavigation-link #{'u-notice' if notice}"
+    link_to path, class: class_name do
       link_text = link_text.prepend "#{icon(icon_name)} " if icon_name
       link_text.html_safe
     end
   end
 
-  def dashboard_secondary_collection_link(singular_name, collection, path, plural = nil)
+  def dashboard_secondary_link(link_text, path, icon_name: nil, notice: false)
     link = ' ('
-    link << dashboard_collection_link(singular_name, collection, path, plural)
+    link << dashboard_navigation_link(link_text, path, icon_name, notice)
     link << ')'
   end
-
-  def dashboard_counter_link(collection, path)
-    link_to collection.size, path, class: "DashboardNavigation-link u-notice"
+  
+  def dashboard_collection_link(singular_name, collection, path, icon_name: nil, notice: false, plural: nil)
```

Comment:
I would go with

```ruby
def dashboard_collection_link(singular_name, collection, path, options = {})
```

Shorter, more open and fixes the unneeded defaults to `nil` and `false`.